### DISCUSSION TITLE
How to update link to forward when sharing data docs into Azure Blob Storage

### DISCUSSION THREAD

**Discussion by guido-vries** [great-expectations/great_expectations#5850] [Support]
I have updated my environment to store the data docs into a Azure Blob Storage container. The link that is generated is incorrect however. It automatically refers to 
https://<myenvironment>.blob.core.windows.net/ while the correct referral is the https://<myenvironment>.z6.web.core.windows.net/

How can i overwrite the correct url?

**Comment by kyleaton** [comment]
Hi @ActivateConsulting, 

We can't currently do this, but we will be discussing internally if this is something we want to support. 

Would you be interested in making a contribution here? 

Thanks,
Kyle

**Comment by guido-vries** [comment]
Sure! Not sure if there is anything I can do, but do let me know!

Verstuurd vanaf mijn iPhone

Op 21 sep. 2022 om 17:48 heeft Kyle Eaton ***@***.***> het volgende geschreven:

﻿

Hi @ActivateConsulting<https://github.com/ActivateConsulting>,

We can't currently do this, but we will be discussing internally if this is something we want to support.

Would you be interested in making a contribution here?

Thanks,
Kyle
